src/broker/broker_config.c

```diff
@@ -350,6 +432,41 @@ broker_config_read_internal (const char *conf_file, T_BROKER_INFO * br_info, int
       return -1;
     }
 
+  /* Validate broker keywords, if keyword is not in list terminate */
+  for (i = 0; i < ini->size; i++)
+    {
+      char *key;
+      bool found = false;
+
+      if (ini->key[i] == NULL)
+        {
+          continue;
+        }
+
+        key = strchr (ini->key[i], ':');
```

Comment:
Codes of lines 446 to 467 should be aligned with lines 441.

Reply:
Thank you for your comment. The indentation was fixed.